components/activity_and_insights/insights/top_dms_and_new_members/top_dms_item/top_dms_item.tsx

```diff
@@ -0,0 +1,86 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+import React, {memo, useCallback} from 'react';
+import {useSelector} from 'react-redux';
+import {FormattedMessage} from 'react-intl';
+import {Link} from 'react-router-dom';
+
+import {getTeammateNameDisplaySetting} from 'mattermost-redux/selectors/entities/preferences';
+
+import {UserProfile} from '@mattermost/types/users';
+import {TopDM} from '@mattermost/types/insights';
+import {Team} from '@mattermost/types/teams';
+
+import {displayUsername} from 'mattermost-redux/utils/user_utils';
+
+import OverlayTrigger from 'components/overlay_trigger';
+import Avatar from 'components/widgets/users/avatar';
+import Tooltip from 'components/tooltip';
+
+import {imageURLForUser} from 'utils/utils';
+import Constants from 'utils/constants';
+
+import './../../../activity_and_insights.scss';
+
+type Props = {
+    dm: TopDM;
+    barSize: number;
+    team: Team;
+}
+
+const TopDMsItem = ({dm, barSize, team}: Props) => {
+    const teammateNameDisplaySetting = useSelector(getTeammateNameDisplaySetting);
+
+    const tooltip = useCallback((messageCount: number) => {
+        return (
+            <Tooltip
+                id='total-messages'
+            >
+                <FormattedMessage
+                    id='insights.topChannels.messageCount'
+                    defaultMessage='{messageCount} total messages'
+                    values={{
+                        messageCount,
+                    }}
+                />
+            </Tooltip>
+        );
+    }, []);
+
+    return (
+        <div className='top-dms-item'>
+            <Avatar
+                url={imageURLForUser(dm.second_participant.id, dm.second_participant.last_picture_update)}
+                size={'xl'}
+            />
+            <div className='dm-info'>
+                <Link
+                    className='dm-name'
+                    to={`/${team.name}/messages/@${dm.second_participant.username}`}
```

Comment:
I was curious about the behaviour if the second user doesn't exist in a team? Should I do this check in backend?

Reply:
That's a good question, if the user is not part of the team should they appear in the top dms in that team? Or are the DMs team agnostic?

cc @matthewbirtch

Reply:
DMs are team agnostic anyway, so I would say we don't factor in the team at all in to this widget